modules/mcc/include/opencv2/mcc/ccm.hpp

```diff
@@ -0,0 +1,493 @@
+                                Macbeth(Macbeth ColorChecker) ;
+                                Vinyl(DKK ColorChecker) ;
+                                DigitalSG(DigitalSG ColorChecker with 140 squares);\n
+                        type: enum CONST_COLOR;\n
+        */
+    ColorCorrectionModel(Mat src, CONST_COLOR constcolor);
+     /** @brief Color Correction Model
+            @param src detected colors of ColorChecker patches;\n
+                        the color type is RGB not BGR, and the color values are in [0, 1];
+                        type is cv::Mat;
+            @param colors the reference color values,the color values are in [0, 1].\n
+                        type: cv::Mat
+            @param ref_cs the corresponding color space
+                        NOTICE: For the list of color spaces supported, see the notes above;\n
+                        If the color type is some RGB, the format is RGB not BGR;\n
+                        type:enum COLOR_SPACE;
+        */
+    ColorCorrectionModel(Mat src, Mat colors, COLOR_SPACE ref_cs);
+    /** @brief Color Correction Model
+            @param src detected colors of ColorChecker patches;\n
+                        the color type is RGB not BGR, and the color values are in [0, 1];
+                        type is cv::Mat;
+            @param colors the reference color values,the color values are in [0, 1].\n
+                        type: cv::Mat
+            @param ref_cs the corresponding color space
+                        NOTICE: For the list of color spaces supported, see the notes above;\n
+                        If the color type is some RGB, the format is RGB not BGR;\n
+                        type:enum COLOR_SPACE;
+            @param colored mask of colored color
+        */
+    ColorCorrectionModel(Mat src, Mat colors, COLOR_SPACE ref_cs, Mat colored);
+
+        /** @brief set ColorSpace
+            @param cs_ the absolute color space that detected colors convert to;\n
+            NOTICE: it should be some RGB color space;\n
+                    For the list of RGB color spaces supported, see the notes above;
+            type: enum COLOR_SPACE;\n
+            default: sRGB;
+        */
+    CV_WRAP void setColorSpace(COLOR_SPACE cs_);
+        /** @brief set ccm_type
+            @param ccm_type :the shape of color correction matrix(CCM);\n
+            Supported list: "CCM_3x3"(3x3 matrix);"CCM_4x3"( 4x3 matrix);\n
+            type: enum CCM_TYPE;\n
+            default: CCM_3x3;\n
+        */
+    CV_WRAP void setCCM(CCM_TYPE ccm_type);
+    /** @brief set Distance
+        @param distance the type of color distance;\n
+            Supported list:"CIE2000"; "CIE94_GRAPHIC_ARTS";"CIE94_TEXTILES";
+                "CIE76";
+                "CMC_1TO1";
+                "CMC_2TO1";
+                "RGB" (Euclidean distance of rgb color space);
+                "RGBL" () Euclidean distance of rgbl color space);\n
+            type: enum DISTANCE_TYPE;\n
```

Comment:
Perhaps it is better to avoid "type: " information for parameters.
This is duplicated information in general

Reply:
OK, we removed "type" notice.